code/core/src/phone/java/com/adobe/marketing/mobile/services/SQLiteDataQueue.java

```diff
@@ -349,4 +339,62 @@ private void createTableIfNotExists() {
                         "createTableIfNotExists - Error creating/accessing table (%s)  ",
                         TABLE_NAME));
     }
+
+    /**
+     * Adds a a Table with name {@link #TABLE_NAME}, if not already exists in database at path
+     * {@link #databasePath}.
+     */
```

Comment:
Add a comment here stating the caller must hold the `dbMutex` when calling this function.